src/ai/backend/client/cli/vfolder.py

```diff
@@ -430,7 +430,7 @@ def cp(filenames):
 
 @vfolder.command()
 @click.argument("name", type=str)
-@click.argument("path", type=str)
+@click.argument("paths", type=str, nargs=-1)
```

Comment:
maybe should I change paths to path?

Reply:
Since this is an argument name, not the option name, I think it is safe to change the argument name without breaking existing client CLI users.